lib/ret/asset.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Ret.Asset do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Ret.{Asset}
+
+  @schema_prefix "ret0"
+  @primary_key {:asset_id, :id, autogenerate: true}
+
+  schema "assets" do
+    field(:asset_sid, :string)
+    field(:name, :string)
+    field(:type, :string)
+    belongs_to(:account, Ret.Account, references: :account_id)
+    belongs_to(:asset_owned_file, Ret.OwnedFile, references: :owned_file_id)
+    many_to_many(:projects, Ret.Project, join_through: Ret.ProjectAsset, join_keys: [ asset_id: :asset_id, project_id: :project_id], on_replace: :delete)
```

Comment:
you should add a `state` column (and check it in the various queries for assets) similar to how we do with scenes/listings/etc

Reply:
it can just have one possible value for now, `:active`